src/components/Sum.jsx

```diff
@@ -1 +1,13 @@
-// export function Sum() {}
+import React from 'react';
+
+const Sum = (props) => {
```

Comment:
```suggestion
export const Sum = (props) => {
```

you need the named export to pass the tests 😉